src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -523,8 +507,8 @@ private void ProcessProjectChange(Solution solution, ProjectId projectId)
             }
 
             // Project was changed in some way.  Let's go find the set of installed packages for it.
-            var dteProject = _workspace.TryGetDTEProject(projectId);
-            if (dteProject == null)
+            var projectGuid = _workspace.GetProjectGuid(projectId);
+            if (projectGuid == Guid.Empty)
             {
                 // Don't have a DTE project for this project ID.  not something we can query NuGet for.
```

Comment:
❔ Are we still using DTE for this?

Reply:
No.  GetProjectGuid itself uses data that is safe to acquire from any thread.